app/decorators/concerns/member_of_collections_display.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+module MemberOfCollectionsDisplay
```

Comment:
This makes me pretty uncomfortable. I'm not willing to stop the PR over it, but I wonder if we need a different pattern.

Reply:
(I don't know what that pattern is - maybe some sort of global property rendering registration?)

Reply:
In fact, maybe it's better to just add `member_of_collections` to the default decorator and have the individual decorators add it to `display_attributes` if they need it?

Reply:
I think the default decorator is a good spot for `member_of_collections`.